src/analysis/validation.jl

```diff
@@ -5,11 +5,24 @@ Perform EPANET hydraulic simulation (via WNTR) and compute timeseries of flows a
 
 using DataFrames
 
+
+function _get_wntr_node_attribute(wntr_simulation, node_name::String, attribute::String)
+    values = PyCall.getproperty(wntr_simulation.node[attribute], node_name).values[1:end-1]
```

Comment:
Is there a reason you used `Pycall.getproperty`? Just for clarity because it is a Python object?
`values = wntr_simulation.node[attribute][node_name].values[1:end-1]`
worked for me with no errors or warnings.  I don't have a strong opinion here; just curious.

Reply:
I assumed this is what PyCall wanted since it was specified in the warning. Maybe if the attributes are variable, `getproperty` isn't needed.